[2025-04-14 12:21:17,595] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 12:25:09,307] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 12:25:11,321] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 12:36:43,700] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 12:36:46,003] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 12:37:32,087] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:02:17,890] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:02:21,345] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:02:23,323] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:03:06,281] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:03:09,052] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:03:42,886] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:03:43,573] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:03:44,101] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:03:44,112] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:04:29,610] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:04:33,495] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:04:35,233] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:07:32,264] [ERROR] [database] 检查StockInfo表状态失败: cannot perform operation: another operation is in progress
[2025-04-14 14:07:34,838] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:07:34,839] [ERROR] [database] 批量插入失败: cannot perform operation: another operation is in progress
[2025-04-14 14:07:34,840] [ERROR] [database] 获取所有股票信息失败: cannot perform operation: another operation is in progress
[2025-04-14 14:07:34,845] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:07:44,613] [ERROR] [database] 检查StockInfo表状态失败: cannot perform operation: another operation is in progress
[2025-04-14 14:07:47,526] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:07:47,527] [ERROR] [database] 批量插入失败: cannot perform operation: another operation is in progress
[2025-04-14 14:07:47,527] [ERROR] [database] 获取所有股票信息失败: cannot perform operation: another operation is in progress
[2025-04-14 14:07:47,535] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:08:15,847] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:08:21,520] [ERROR] [database] 检查StockInfo表状态失败: cannot perform operation: another operation is in progress
[2025-04-14 14:08:24,691] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:08:24,692] [ERROR] [database] 批量插入失败: cannot perform operation: another operation is in progress
[2025-04-14 14:08:24,693] [ERROR] [database] 获取所有股票信息失败: cannot perform operation: another operation is in progress
[2025-04-14 14:08:24,702] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:12:10,986] [ERROR] [database] 检查StockInfo表状态失败: cannot perform operation: another operation is in progress
[2025-04-14 14:12:13,523] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:12:13,524] [ERROR] [database] 批量插入失败: cannot perform operation: another operation is in progress
[2025-04-14 14:12:13,525] [ERROR] [database] 获取所有股票信息失败: cannot perform operation: another operation is in progress
[2025-04-14 14:12:13,529] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:13:46,922] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:13:57,993] [ERROR] [database] 检查StockInfo表状态失败: cannot perform operation: another operation is in progress
[2025-04-14 14:14:01,302] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:14:01,303] [ERROR] [database] 批量插入失败: cannot perform operation: another operation is in progress
[2025-04-14 14:14:01,303] [ERROR] [database] 获取所有股票信息失败: cannot perform operation: another operation is in progress
[2025-04-14 14:14:01,313] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:20:46,190] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:21:32,867] [ERROR] [database] 检查StockInfo表状态失败: cannot perform operation: another operation is in progress
[2025-04-14 14:21:35,696] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:21:35,697] [ERROR] [database] 批量插入失败: cannot perform operation: another operation is in progress
[2025-04-14 14:21:35,697] [ERROR] [database] 获取所有股票信息失败: cannot perform operation: another operation is in progress
[2025-04-14 14:21:35,706] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:26:33,298] [ERROR] [database] 检查StockInfo表状态失败: cannot perform operation: another operation is in progress
[2025-04-14 14:26:38,667] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:26:38,669] [ERROR] [database] 批量插入失败: cannot perform operation: another operation is in progress
[2025-04-14 14:26:38,670] [ERROR] [database] 获取所有股票信息失败: cannot perform operation: another operation is in progress
[2025-04-14 14:26:38,687] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:32:51,622] [ERROR] [database] 检查StockInfo表状态失败: cannot perform operation: another operation is in progress
[2025-04-14 14:32:54,253] [ERROR] [database] 批量插入失败: Event loop is closed
[2025-04-14 14:32:54,253] [ERROR] [database] 获取所有股票信息失败: Event loop is closed
[2025-04-14 14:32:54,259] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:33:25,423] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:34:47,662] [ERROR] [database] 检查StockInfo表状态失败: cannot perform operation: another operation is in progress
[2025-04-14 14:34:59,538] [ERROR] [database] 批量插入失败: Event loop is closed
[2025-04-14 14:34:59,539] [ERROR] [database] 获取所有股票信息失败: Event loop is closed
[2025-04-14 14:34:59,551] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:41:39,375] [ERROR] [database] 检查StockInfo表状态失败: Event loop is closed
[2025-04-14 14:41:42,136] [ERROR] [database] 批量插入失败: Event loop is closed
[2025-04-14 14:41:42,136] [ERROR] [database] 获取所有股票信息失败: Event loop is closed
[2025-04-14 14:42:32,151] [ERROR] [database] 检查StockInfo表状态失败: Event loop is closed
[2025-04-14 14:42:34,695] [ERROR] [database] 批量插入失败: Event loop is closed
[2025-04-14 14:42:34,695] [ERROR] [database] 获取所有股票信息失败: Event loop is closed
[2025-04-14 14:48:57,062] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 14:50:22,917] [ERROR] [database] 检查StockInfo表状态失败: cannot perform operation: another operation is in progress
[2025-04-14 14:50:32,057] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:50:32,061] [ERROR] [database] 批量插入失败: cannot perform operation: another operation is in progress
[2025-04-14 14:50:32,061] [ERROR] [database] 获取所有股票信息失败: cannot perform operation: another operation is in progress
[2025-04-14 14:50:32,075] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 14:58:08,120] [ERROR] [database] 检查StockInfo表状态失败: Event loop is closed
[2025-04-14 14:58:10,854] [ERROR] [database] 批量插入失败: Event loop is closed
[2025-04-14 14:58:10,854] [ERROR] [database] 获取所有股票信息失败: Event loop is closed
[2025-04-14 15:15:45,052] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 15:18:31,618] [ERROR] [database] 检查StockInfo表状态失败: cannot perform operation: another operation is in progress
[2025-04-14 15:18:34,986] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 15:18:34,987] [ERROR] [database] 批量插入失败: cannot perform operation: another operation is in progress
[2025-04-14 15:18:34,987] [ERROR] [database] 获取所有股票信息失败: cannot perform operation: another operation is in progress
[2025-04-14 15:18:34,993] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 15:34:20,088] [INFO] [database] 数据库表结构初始化完成
[2025-04-14 15:37:49,526] [ERROR] [database] 检查StockInfo表状态失败: cannot perform operation: another operation is in progress
[2025-04-14 15:37:57,160] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
[2025-04-14 15:37:57,193] [ERROR] [database] 批量插入失败: cannot perform operation: another operation is in progress
[2025-04-14 15:37:57,195] [ERROR] [database] 获取所有股票信息失败: cannot perform operation: another operation is in progress
[2025-04-14 15:37:57,226] [ERROR] [base_events] Future exception was never retrieved
future: <Future finished exception=ConnectionDoesNotExistError('connection was closed in the middle of operation')>
asyncpg.exceptions.ConnectionDoesNotExistError: connection was closed in the middle of operation
